openshift/scripts/common.sh

```diff
@@ -37,10 +37,12 @@ source "$(dirname ${0})/envars"
 
 # Verify login
 #
-$(oc whoami &>/dev/null) || {
-  echo -e "\nPlease verify oc login\n"
-  exit 1
-}
+if [ -z "${SKIP_OC_LOGIN:-}" ]; then
+  $(oc whoami &>/dev/null) || {
+    echo -e "\nPlease verify oc login\n"
+    exit 1
+  }
+fi
```

Comment:
Provide a way to skip logging into Openshift as this is not yet supported from github actions